loom.config

```diff
@@ -2,14 +2,14 @@
   "sdk_version": "1.1.4432",
   "executable": "Main.loom",
   "display": {
-    "width": 1024,
-    "height": 768,
+    "width": 1080,
+    "height": 1920,
```

Comment:
On my Mac it just fits to the available space. So the actual res I dev on is different.